web-animations-2/Overview.bs

```diff
@@ -3028,21 +3049,51 @@ with which it is associated is updated.
 
 </ins>
 
-<h3 id="the-animatable-interface">The <code>Animatable</code> interface</h3>
+The <code>Animatable</code> interface mixin {#the-animatable-interface-mixin}
+----------------------------------------
 
-<div class="methods">
+The {{KeyframeAnimationOptions}} dictionary interface
+is modified to add the following members:
 
-:   <dfn method for=Animatable lt="getAnimations()">
-    sequence&lt;Animation&gt; getAnimations()</dfn>
-::  Add:
+<pre class='idl'>
+dictionary TimelineRangeOffset {
+  CSSOMString? rangeName;
+  CSSNumericValue offset;
+};
 
-    > If this object is the [=effect target/target element=] of two or more
-    > <a>animation effects</a> which are associated with the
-    > same <a>animation</a>, the corresponding {{Animation}}
-    > object will still only appear in the returned list once.
```

Comment:
This seems to have been unintentionally removed.

Reply:
Thanks, fixed.